<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemCreator</class>
 <widget class="QMainWindow" name="ItemCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VigiLost</string>
  </property>
  <property name="windowIcon">
   <iconset theme="VigiIcon">
    <normaloff>vigi.png</normaloff>vigi.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelClave">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Buscar palabra clave:</string>
        </property>
        <property name="buddy">
         <cstring>dataBuscar</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dataBuscar"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="radioPerdidos">
          <property name="minimumSize">
           <size>
            <width>123</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>124</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>Objetos Perdidos</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioEncontrados">
          <property name="minimumSize">
           <size>
            <width>124</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>124</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>Objetos Encontrados</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="actualizarBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Actualizar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item alignment="Qt::AlignVCenter">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>label objetos</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <column>
       <property name="text">
        <string>Objeto</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Propietario</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color/Dibujo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Encontrado(a) en</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Otros</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Encontrado por</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAcerca_del_Programa">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="actionAcerca_de_VigiLost"/>
   </widget>
   <addaction name="menuAcerca_del_Programa"/>
  </widget>
  <action name="actionAcerca_de_VigiLost">
   <property name="text">
    <string>Acerca de VigiLost</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dataBuscar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
